### PR TITLE
feat(Card): multiple card variations

### PR DESCRIPTION
**Pull request by ghlost**
Include many of the variations except for featured

### CODE REVIEW

File: packages/twig/src/patterns/components/card/card.twig [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      {% if eyebrow %}
+        <p class="{{prefix}}--card--eyebrow">{{eyebrow}}</p>
+      {% endif %}
+      {% if title %}
+        <h5 class="{{prefix}}--card--title">{{title}}</h5>
+      {% endif %}
+      {% if intro %}
+        <p class="{{prefix}}--card--intro">{{intro}}</p>
+      {% endif %}
+      {% if date %}
```
Comment (johnpauldavis) [inline packages/twig/src/patterns/components/card/card.twig:23]: Can you make this the `<time>` tag and also output the UNIX `datetime` attribute?

File: packages/twig/src/patterns/components/card/card.twig [hunk 2]
```diff
@@ -0,0 +1,95 @@
+{#
+  CARD COMPONENT
+#}
+<script>console.log({{ _context | json_encode | raw }});</script>
```
Comment (johnpauldavis) [inline packages/twig/src/patterns/components/card/card.twig:4]: Don't forget to remove this

File: packages/twig/src/patterns/components/card/card.twig [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      {% if title %}
+        <h5 class="{{prefix}}--card--title">{{title}}</h5>
+      {% endif %}
+      {% if intro %}
+        <p class="{{prefix}}--card--intro">{{intro}}</p>
+      {% endif %}
+      {% if date %}
+        <p class="{{prefix}}--card--date">{{date}}</p>
+      {% endif %}
+      {% if eventdate %}
```
Comment (johnpauldavis) [inline packages/twig/src/patterns/components/card/card.twig:26]: Same here

File: packages/twig/src/patterns/components/cardgroup/cardgroup.twig [hunk 4]
```diff
@@ -0,0 +1,31 @@
+{#
+  CARDGROUP COMPONENT
+#}
+<script>console.log({{ _context | json_encode | raw }});</script>
```
Comment (johnpauldavis) [inline packages/twig/src/patterns/components/cardgroup/cardgroup.twig:4]: Also remove this one

**Review by johnpauldavis — CHANGES_REQUESTED** [review]
Nice work @ghlost - just a few minor things in the comments

File: packages/twig/src/patterns/components/card/card.twig (hunk 3 above)
Comment (ghlost) [reply packages/twig/src/patterns/components/card/card.twig]: I don't this is actually a date necessarily but it looks like some details so i changed the param to `eventdetails` instead.

File: packages/twig/src/patterns/components/card/card.twig (hunk 2 above)
Comment (ghlost) [reply packages/twig/src/patterns/components/card/card.twig]: Done

File: packages/twig/src/patterns/components/card/card.twig (hunk 1 above)
Comment (ghlost) [reply packages/twig/src/patterns/components/card/card.twig:22]: Yes good point.

File: packages/twig/src/patterns/components/cardgroup/cardgroup.twig (hunk 4 above)
Comment (ghlost) [reply packages/twig/src/patterns/components/cardgroup/cardgroup.twig]: yes

**Pull request merged**